testnets/zenrocktestnet/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "zenrocktestnet",
+    "assets": [
+      {
+        "description": "The native token of zenrock",
+        "denom_units": [
+          {
+            "denom": "urock",
+            "exponent": 0,
+            "aliases": []
+          },
+          {
+            "denom": "rock",
+            "exponent": 6,
+            "aliases": []
+          }
+        ],
+        "base": "urock",
+        "name": "Zenrock",
+        "display": "rock",
+        "symbol": "ROCK",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/zenrocktestnet/images/zenrock.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/zenrocktestnet/images/zenrock.svg"
+        },
+        "keywords": [
+          "mpc"
+        ],
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/zenrocktestnet/images/zenrock.png",
```

Comment:
The png and svg, because they are visually distinct, would have to go into individual objects within the array, and only one can be added to the logo_URIs

Reply:
Thank you. Made the requested changes. Can you check again?